apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformer.java

```diff
@@ -1068,15 +1113,30 @@ private RdaFissDiagnosisCode transformMessageImpl(
         to::setDiagPoaInd);
     transformer.copyOptionalString(
         namePrefix + RdaFissDiagnosisCode.Fields.bitFlags,
-        1,
+        0,
         4,
         from::hasBitFlags,
         from::getBitFlags,
         to::setBitFlags);
     to.setLastUpdated(now);
+
+    // At least one of these two fields must have a value for the object to be valid.
+    transformer.validateAtLeastOneIsPresent(
```

Comment:
Thanks for having the foresight for including the option to define transformations for multiple fields!